__tests__/api.test.js

```diff
@@ -28,4 +28,12 @@ describe("/api/categories", () => {
         });
       });
   });
+  test("GET request responds with status 404 and error message", () => {
```

Comment:
This should be in its own describe block as it is not realted to /api/categories